snippets/fer1_annoted.js

```diff
@@ -54,5 +56,51 @@ var defMenu = new msa.menu.defaultmenu({
 });
 defMenu.render();
 
+// pin features
+const pinnedFeatures = [
```

Comment:
This has similar datastructure as regular features.
The difference is that regular features map features by sequence ID, but pinned features don't have them since there's no sequence associated with it.

i.e,
regular features:
```
seqs: {
  'entity001': [feature1, feature2],
  'entity002': [feature3, feature4],
}
```
pinned features:
```
[
  feature1, 
  feature2,
]
```